inc/saiacl.h

```diff
@@ -221,11 +221,15 @@ typedef enum _sai_acl_table_attr_t
     
     /** READ-ONLY */
 
+    /** Switch Object ID [sai_object_id_t] (CREATE_ONLY),
+     * Default SAI_NULL_OBJECT_ID */
```

Comment:
agree that use SAI_NULL_OBJECT_ID to refer the default switch id is not a good idea. The proposal is to make this attribute as mandatory_on_create, the use must supply a value, which is returned by sai_create_switch() API.